test/unit/test_manifest_download.py

```diff
@@ -0,0 +1,75 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+from mock import patch
+
+from hca.dss import DSSClient
+
+
+def _touch_file(*args, **kwargs):
+    path = args[2] if len(args) > 2 else args[0]
+    try:
+        os.makedirs(os.path.split(path)[0])
+    except OSError:
+        # dirs already exist
```

Comment:
Not always true, I think:

```
>>> import os
>>> os.makedirs("/x")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/x'
>>> os.makedirs("/")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
IsADirectoryError: [Errno 21] Is a directory: '/'
>>> os.makedirs("/tmp/foo")
>>> os.makedirs("/tmp/foo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/tmp/foo'
```

Reply:
Good point. I know on 2.7 FileExistsError doesn't exist, but I can still check the error string in a portable way.